src/decorator/options/ColumnJsonOptions.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Column options for enum-typed columns.
```

Comment:
Wrong comment.

Reply:
Hmm, this was a copy-paste from ColumnHstoreOptions. I guess it doesn't make much sense there, either. Will ammend to 'options for JSON-typed'